HopsanGUI/Dialogs/OptimizationScriptWizard.cpp

```diff
@@ -1496,15 +1504,15 @@ void OptimizationScriptWizard::loadConfiguration()
         if(idx > -1) //found!
         {//Lägg till variabel i XML -> compname, portname, varname, ska vara i mSelectedVariables
             mpFunctionsComboBox->setCurrentIndex(idx);
-            addObjectiveFunction(idx, optSettings.mObjectives.at(i).mWeight, optSettings.mObjectives.at(i).mNorm, optSettings.mObjectives.at(i).mExp, optSettings.mObjectives.at(i).mVariableInfo, optSettings.mObjectives.at(i).mData);
+            addObjectiveFunction(idx, optSettings.mObjectives.at(i).mWeight, optSettings.mObjectives.at(i).mNorm, optSettings.mObjectives.at(i).mExp, optSettings.mObjectives.at(i).mVariableInfo, optSettings.mObjectives.at(i).mData, true);
```

Comment:
I realize that this code was already here, but I would prefer to get the mSettings.mObjectives.at(i) once, and create a reference to it. Then use rhe reference instead of the above call multiple times